src/store.js

```diff
@@ -0,0 +1 @@
+export default function store() {}
```

Comment:
I guess you are not done with redux... However, I recommend to not add it in this PR, as that would be very hard to separate from the other changes and correspondingly it would increase the difficulty of reviewing this.